### PR TITLE
Bump PHP dependencies

### PR DESCRIPTION
**Pull request by MusikAnimal**
Also adds symfony/mime, which was apparently unbundled from some other
Symfony package.

Bug: T262464

### CODE REVIEW

**Review by samwilson — CHANGES_REQUESTED** [review]
The ULS dialog isn't working for me with this patch and shows the error "Uncaught TypeError: targetLangButton.$element.uls is not a function". It might be because I'm not building things correctly, but I don't think so. The error I'm getting the same one described in https://phabricator.wikimedia.org/T233182

Also, webpack is giving the following warning:

> WARNING   Webpack Encore requires version ^6.2.0 of less-loader, but your version (4.1.0) is too old. > The related feature will probably *not* work correctly.

**Comment by MusikAnimal** [comment]
> The ULS dialog isn't working for me with this patch and shows the error "Uncaught TypeError: targetLangButton.$element.uls is not a function". It might be because I'm not building things correctly, but I don't think so. The error I'm getting the same one described in https://phabricator.wikimedia.org/T233182
> 
> Also, webpack is giving the following warning:
> 
> > WARNING   Webpack Encore requires version ^6.2.0 of less-loader, but your version (4.1.0) is too old. > The related feature will probably _not_ work correctly.

Ah, I obviously didn't test the language selector! I tried all sorts of combinations of packages, such as locking the old version of jQuery, ULS, OOUI and Webpack, but was still running into this issue. So for now this PR is just for PHP dependencies. Note also have a lot of deprecation notices now, but I assume those can be ironed out in a future PR.

File: composer.json [hunk 1]
```diff
@@ -31,7 +32,7 @@
         "symfony/validator": "^4.1",
         "symfony/web-link": "^4.1",
         "symfony/yaml": "^4.1",
-        "wikimedia/toolforge-bundle": "^0.18"
+        "wikimedia/toolforge-bundle": "^0.19"
```
Comment (samwilson) [inline composer.json:35]: This can be `^1.0` now. :)

**Review by samwilson — COMMENTED** [review]
I don't know if it matters, but this can be rebased.

**Pull request merged**